src/Hazelcast.Net/Core/TimeSpanExtensions.cs

```diff
@@ -57,5 +57,10 @@ public static int TimeoutMilliseconds(this TimeSpan timeSpan, int defaultTimeout
             var timeout = (int) timeSpan.TotalMilliseconds;
             return timeout > 0 ? timeout : timeout < 0 ? infiniteTimeout : defaultTimeout;
         }
+
+        public static long ToTotalMilliseconds(this TimeSpan timeSpan)
```

Comment:
Can we rename `TotalRoundedMilliseconds` or `TotalMillisecondsRounded` so that it looks like the other methods (none have the `ToXxx` prefix? And add documentation?

Reply:
And, returning `(long) timeSpan.TotalMilliseconds` would be way faster, avoiding a division.